tests/io/test_resume_kti.py

```diff
@@ -0,0 +1,112 @@
+from scaaml.io.resume_kti import create_resume_kti, ResumeKTI
+import numpy as np
+import os
+
+KT_FILENAME = 'key_text_pairs.txt'
+PROGRESS_FILENAME = 'progress_pairs.txt'
+KEYS = np.array([3, 1, 4, 1, 5, 9, 0, 254, 255, 0])
+TEXTS = np.array([2, 7, 1, 2, 8, 0, 255, 254, 1, 0])
+SHARD_LENGTH = 2
+
+
+def test_save_and_load(tmp_path):
+    # Check that the files do not exist now
+    assert not os.path.isfile(tmp_path / KT_FILENAME)
+    assert not os.path.isfile(tmp_path / PROGRESS_FILENAME)
+    create_resume_kti(keys=KEYS,
+                      texts=TEXTS,
+                      shard_length=SHARD_LENGTH,
+                      kt_filename=tmp_path / KT_FILENAME,
+                      progress_filename=tmp_path / PROGRESS_FILENAME)
+    # Check that the files exist now
+    assert os.path.isfile(tmp_path / KT_FILENAME)
+    assert os.path.isfile(tmp_path / PROGRESS_FILENAME)
+
+    resume_kti = ResumeKTI(kt_filename=tmp_path / KT_FILENAME,
```

Comment:
This line becomes obsolete now that `create_resume_kti()` directly constructs the object. Merge with line 16

Reply:
Good point